doc/en/example/simple.rst

```diff
@@ -895,29 +895,35 @@ here is a little example implemented via a local plugin:
 
     import pytest
 
+    your_unique_key = StashKey[Dict[str, bool]]()
+
 
     @pytest.hookimpl(tryfirst=True, hookwrapper=True)
     def pytest_runtest_makereport(item, call):
         # execute all other hooks to obtain the report object
         outcome = yield
         rep = outcome.get_result()
 
-        # set a report attribute for each phase of a call, which can
+        # store test results for each phase of a call, which can
         # be "setup", "call", "teardown"
-
-        setattr(item, "rep_" + rep.when, rep)
+        if your_unique_key not in item.stash:
+            item.stash[your_unique_key] = {rep.when: rep.passed}
```

Comment:
The previous code stored the entire report, I think that's a bit more generally useful, so would keep that instead of storing just `passed`.

Reply:
Agreed.
e20dfb6